src/app/rpmostree-builtin-status.cxx

```diff
@@ -1248,15 +1252,21 @@ print_history_entry (RORHistoryEntry  *entry,
         }
 
       json_builder_set_member_name (builder, "deployment-create-timestamp");
-      json_builder_add_int_value (builder, entry->deploy_timestamp);
+      json_builder_add_int_value (builder, entry.deploy_timestamp);
       json_builder_set_member_name (builder, "deployment-create-command-line");
```

Comment:
Need to delete this line now that it's folded into the if-statement.

Reply:
Ah yep thanks, it's history now.